app/src/main/java/br/com/dionataferraz/vendas/data/remote/RetrofitNetworkClient.kt

```diff
@@ -0,0 +1,55 @@
+package br.com.dionataferraz.vendas.data.remote
+
+import br.com.dionataferraz.vendas.BuildConfig
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import java.util.concurrent.TimeUnit
+
```

Comment:
Junto dessas outras duas
```suggestion
import com.squareup.moshi.adapters.Rfc3339DateJsonAdapter
import java.util.Date

```